dags/veda_data_pipeline/groups/processing_tasks.py

```diff
@@ -12,6 +13,21 @@
 def log_task(text: str):
     logging.info(text)
 
+@task()
+def extract_discovery_items_from_payload(ti, **kwargs):
```

Comment:
`ti` is a reserved parameter (kwargs already includes `ti`). We need to change this function to either 
```python
@task()
def extract_discovery_items_from_payload(ti):
    discovery_items = ti.dag_run.conf.get("discovery_items")
    return discovery_items
```
Or
```python
@task()
def extract_discovery_items_from_payload(**kwargs):
    ti = kwargs.get('ti')
    discovery_items = ti.dag_run.conf.get("discovery_items")
    return discovery_items
```